website/views/static/login-modal.jade

```diff
@@ -26,14 +26,14 @@ script(id='modals/login.html', type='text/ng-template')
     button.close(type='button', ng-click='$close()') ×
     h4.modal-title=env.t('loginAndReg')
   .modal-body(ng-controller='AuthCtrl')
-    a.zocial.facebook(alt=env.t('loginFacebookAlt'), ng-click='socialLogin("facebook")')=env.t('loginFacebookAlt')
-    //-ul.list-inline
+    //a.zocial.facebook(alt=env.t('loginFacebookAlt'), ng-click='socialLogin("facebook")')=env.t('loginFacebookAlt')
+    ul.list-inline
       li
         a.zocial.icon.facebook(alt=env.t('loginFacebookAlt'), ng-click='socialLogin("facebook")')
       li
         a.zocial.icon.googleplus(alt="Google", ng-click='socialLogin("google")') Google+
```

Comment:
Does this need a label? The Facebook one does not.

Also, it's technically not a google plus login.